docs/user_guide/index.rst

```diff
@@ -490,10 +490,8 @@ Then, follow the pattern of normal startup but specify your function in the :py:
 
 .. code-block:: python
 
-   from seismometer.configuration import ConfigProvider
-   from seismometer.data.loader import loader_factory
+   import seismometer.api as sm
    from seismometer.seismogram import Seismogram
```

Comment:
should we update this line, it feels odd to import Seismogram from a different place than API.